_posts/2020-02-19-js-the-language-meetup.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: "JavaScript The Language:  Async/Await"
+date: 2020-02-19 10:00:00 +0200
+categories: javascript
+author_names: ["Pablo Palacios", "Sergii Paryzhskyi"]
+read_time : 5
+show_related_posts: false
+square_related:
+feature_image: posts/2020-02-19-js-the-language-meetup-async-await/poster.jpg
+---
+
+Yesterday we've hosted a “JavaScript the Language” Meetup at HolidayCheck. This is a monthly meetup where we go into details about specific aspects of JavaScript. The topic of Thursday's meetup was Async/Await.
+
+There is no presentations, but it is a very interactive and hands-on event. Instead of a keynote speaker and slides presentation, the
+guests here are protagonists. As usually it all starts with a socialising, and get to know each other while enjoying snacks and drinks:
+
+<img src="/img/posts/2020-02-19-js-the-language-meetup-async-await/socializing.jpg" alt="Socializing" class="centered" />
+
+After having some food and drinks, we start an introduction round. Everything is very informal and its a good start to make people talk. Normally people say where they came from or share experience they've made with JavaScript so far. That was the point as we've realized how broad our audience actually is. Some people had barely started with programming, some others were actively using JavaScript over 15 years. That turned out not to be a problem at all. In order to bring everyone on the same page about async/await, we've started by reminding ourself the general knowledge about this language construct and what it is normally used for. Afterward we've started to collect all the things that people would be interested to know in context of async/await:
+
+<img src="/img/posts/2020-02-19-js-the-language-meetup-async-await/collect-ideas.jpg" alt="Collect ideas" class="centered" />
+
+Process of collecting ideas and questions went quite actively and proposals were coming from participants of all levels. It seems like it took us less than 10 minutes to fill up the flipchart with lots of exciting topics to tackle:
```

Comment:
Maybe something like 'Process of collecting ideas and questions went quite actively and proposals were coming from participants of all levels.' -> 'Participants of all experience levels were actively engaged in proposing ideas and questions to be discussed in more detail.'